README.md

```diff
@@ -136,42 +136,50 @@ plugin. You can turn on/off an option by adding/removing the corresponding flag
-3. Install dependencies using `npm install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.)
-   See the [npm Docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
-4. Enable the corepack utility by running `corepack enable`, as superuser.
-   See the [Corepack homepage](https://github.com/nodejs/corepack).
-5. Run `npm install -g @vscode/vsce`.
+3. Run `npm install -g @vscode/vsce`, as superuser.
+4. Run `npm add -g @vscode/vsce`.
+
+#### Building extension
 
-To start working on this project, perform the following steps:
 1. Create a working directory for the project and change into it.
-2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git`
-   (adapt the URL to your specific fork and branch). 
+2.  Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git`
 3. Change into the cloned directory `cd checker-framework-vscode`.
-4. Run `npm add -g @vscode/vsce`.
+4. Install dependencies using `npm install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.) See the [npm Docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
+5.  Run `vsce package` to generate a `.vsix` file 
+6.  Run `code --install-extension checker-framework-language-server-0.2.0.vsix` 
+   
+### Test local downloader: 
 
-To compile the downloader locally: 
 1. Change back into your project working directory. 
 2. Clone the downloader in the project directory `git clone https://github.com/eisopux/checker-framework-languageserver-downloader.git`
 3. Change into the downloader folder and run `./gradlew assemble`.
    See file `[README.md](https://github.com/eisopux/checker-framework-languageserver-downloader/blob/master/README.md)` for more details.
 4. Copy `build/libs/checker-framework-languageserver-downloader-all.jar` to the plugin’s folder and rename it to replace the file `checker-framework-languageserver-downloader.jar`
 
-To build the extension from the command line:
-1. In the `checker-framework-vscode` directory, run `vsce package` to generate a `.vsix` file 
-2. Run `code --install-extension checker-framework-language-server-0.2.0.vsix` 
+### Test local language server or checkerframework:
+
+You can use your local checker-framework and language server. Update the appropriate field in `.vscode/settings.json` with your custom path like the following example.
+```json
+{
+    "checker-framework.frameworkPath": "/path/to/custom/framework",
+    "checker-framework.languageServerPath": "/path/to/custom/languageserver.jar"
+}
+```
+
+save the file and restart Visual Studio code.
 
-Under `View -> Extensions` in VS Code check that the 'checker-framework-language-server' is installed. 
-During development, you can simply start the program from the command line and run as a new VS Code instance:
-1. Install dependencies using `npm install`.
```

Comment:
Maybe instead of this, say to follow the "build from source" instructions.

Reply:
@wmdietl I am sorry, since this line is removed, I am not sure which line you mean to add `follow the "build from source" instructions.`

Reply:
This comment is meant for the line `1. Install dependencies using npm install.` That is only one of the steps for building from source. Don't repeat them, just refer to the section.
I see that you added the extension list sentence. I don't see the text about starting a debugging session.